engine/clientdb/dbs.go

```diff
@@ -0,0 +1,115 @@
+package clientdb
+
+import (
+	"database/sql"
+	_ "embed"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/dagger/dagger/engine/slog"
+	_ "modernc.org/sqlite"
+)
+
+type DBs struct {
+	Root string
+}
+
+// CollectGarbageAfter is the time after which a database is considered
+// garbage and can be deleted.
+const CollectGarbageAfter = time.Hour
```

Comment:
Did we change this? I seem to remember in our core sync we discussed that this was quite a bit longer, something like 48 hours? I can't quite remember, but the logic was that we want to avoid the case where a job might accidentally get cleaned up?

Or is that just not possible, and I'm totally misremembering?